content/pt-br/shift-left.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Shift Left
+status: Completed
+category: conceito
+tags: ["metodologia", "", ""]
+---
+
+## O que é
+
+Esquerda em Shift Left se refere aos estágios iniciais no ciclo de desenvolvimento de software,
+pensando no ciclo como uma linha onde os estágios são executados da esquerda para a direita.
+Shift left é a prática de implementar testes, segurança ou outras práticas de desenvolvimento
+no início do ciclo de desenvolvimento de software e não no final.
+
+Embora originalmente usado para se referir ao processo de teste de forma antecipada,
+Shift Left agora também pode ser aplicado para outros aspectos do desenvolvimento de software e [DevOps](/pt-br/devops/), como segurança e implementação (deployment).
+
+## Problema relacionado
+
+Problemas de segurança, erros e defeitos de software podem ser mais difíceis e caros para serem corrigidos
+se eles forem descobertos de forma tardia durante o ciclo de desenvolvimento ou após a implementação,
+principalmente se o software já estiver implementado em produção.
+
+## Como isso ajuda
+
+Adotar a mentalidade de Shift Left para o desenvolvimento de software,
+as equipes podem implementar testes e segurança durante o ciclo de desenvolvimento.
+Pela responsabilidade por testes e segurança ser compartilhada pela equipe de desenvolvimento
+— dos engenheiros de software, qualidade até operações —
+todos tem seu papel em garantir a estabilidade e a segurança de uma aplicação.
+
+Além disso, atuar de forma Shift Left permite a melhoria contínua e
+está em linha com uma abordagem mais [agile](/pt-br/agile-software-development/) do que cascata ao desenvolvimento.
```

Comment:
`agile` can be translated to `ágil` what you think @matheuspaes?

Reply:
Done 😊